sphinx_needs/directives/needextend.py

```diff
@@ -54,20 +53,37 @@ def run(self) -> Sequence[nodes.Node]:
                 f"Filter of needextend must be set. See {env.docname}:{self.lineno}"
             )
 
-        strict = NeedsSphinxConfig(self.env.app.config).needextend_strict
+        needs_config = NeedsSphinxConfig(self.env.app.config)
+        strict = needs_config.needextend_strict
         strict_option: str = self.options.get("strict", "").upper()
         if strict_option == "TRUE":
             strict = True
         elif strict_option == "FALSE":
             strict = False
 
+        modifications = self.options.copy()
+        modifications.pop("strict", None)
+
+        extend_filter = extend_filter.strip()
+        if extend_filter.startswith("<") and extend_filter.endswith(">"):
+            filter_is_id = True
+            extend_filter = extend_filter[1:-1]
```

Comment:
Users provided `<>` or `""` will lead to an empty filter. Is that handled later on?